### ISSUE TITLE
Coordinate frame issue in runAtlasHandControl

### ISSUE THREAD

**Issue by rdeits** [RobotLocomotion/drake#1684]
On master: 

```
Error using Trajectory/inFrame (line 69)
couldn't find a coordinate transform from the trajectory frame atlasFrames.AtlasState+right_atlasFrames.HandState+left_atlasFrames.HandState to the requested
frame atlasPosition+s-model_articulatedPosition+s-model_articulatedPosition

Error in Trajectory/subsref (line 217)
        [varargout{:}] = builtin('subsref',a,s);

Error in Visualizer/playback (line 85)
        xtraj = xtraj.inFrame(obj.getInputFrame);  % try to convert it

Error in runAtlasHandControl (line 150)
  playback(v,traj,struct('slider',true));
```

**Comment by rdeits** [comment]
I don't know how this got through, but it's failing on master for me, and on the branch from #1678

**Comment by RussTedrake** [comment]
I can predict your reaction, @rdeits, but the reason it got through the build servers is probably because they do not test the visualizers (they get a NullVisualizer object which has an empty playback method).  

Assigning Eric, since it was his change.

**Comment by eric-wieser** [comment]
Can you point me to the blame that shows this was me?

**Comment by RussTedrake** [comment]
Ah.  I might be wrong.  I thought it was related to the trajectory frame fixes you passed in earlier.

**Comment by eric-wieser** [comment]
That's not unreasonable to assume - I'll take a look, if I'm able to run that on windows.

**Comment by eric-wieser** [comment]
Yep, no dice - no `gurobi` for me

**Comment by jwnimmer-tri** [comment]
Closing as won't-fix, because #6369 will be removing the relevant software.

If I'm confused and this should in fact stay open, please re-open an clarify the remaining unsolved problem(s) in a new comment here.
